src/plugins/ciu/ciuPlugin.js

```diff
@@ -0,0 +1,73 @@
+const db = require('caniuse-db/data.json');
+const { messageToFactoid } = require('../factoids/factoidsPlugin');
+
+const featureKeys = Object.keys(db.data);
+
+const AGENTS_TO_SHOW = {
+  ie: 'IE',
+  edge: 'Edge',
+  firefox: 'FF',
+  chrome: 'Chrome',
+  opera: 'Opera',
+  safari: 'Safari',
+  ios_saf: 'iOS',
+  android: 'Android',
+};
+
+function getAgentSupportArray(feature) {
+  return Object.entries(AGENTS_TO_SHOW).flatMap(([agent, agentName]) => {
+    let supportedVersions = db.agents[agent].versions.filter(
+      (v) => v && !/[nupax]/g.test(feature.stats[agent][v]),
+    );
+
+    if (supportedVersions.length === 0) {
+      supportedVersions = db.agents[agent].versions.filter(
+        (v) => v && !/[nu]/g.test(feature.stats[agent][v]),
+      );
+    }
+
+    return supportedVersions.length > 0
+      ? [`${agentName}${supportedVersions[0]}`]
```

Comment:
no separator between them?

Reply:
well no need I think

Reply:
wouldn't it be clearer?

Reply:
my thought was that with space, and using | between browsers, `IE 9|FF 66|Chrome 88` is more confusing, the number looks attached to the next word

Reply:
then let's pad the separator with spaces, so it's more readable? :-) or use a comma?

Reply:
but then the grouping between the 4 sections in "description versions supportPercent link" isn't so clear anymore

```
Some feature description IE 9 | FF 66 | Chrome 88 86.74% https://caniuse.com/foo-bar
```

Reply:
`Some feature description (IE 9, FF 66, Chrome 88) 86.74% https://caniuse.com/foo-bar` ?

Reply:
ok, done